frontend/app/sysupport-admin/index/template.hbs

```diff
@@ -1,19 +1,21 @@
 <h2>{{t 'sysupport.admin.projects'}}</h2>
 
 <div class="uk-overflow-auto" uk-scrollspy="target: > div; cls:uk-animation-slide-bottom-small; delay: 50;">
-    {{#cc-data-table headers=headers model=model as |results|}}
+  {{#cc-data-table headers=headers model=model as |results|}}
     {{#each results as |project index|}}
-      <tr>
-      {{#link-to "sysupport-admin.detail" project.id data-test-project=index class="uk-link-reset"}}
-        <td>{{project.name}}</td>
-      {{/link-to}}
+    {{#link-to "sysupport-admin.detail" project.id data-test-project=index class=(if (gte alertTime project.totalTime._milliseconds) "uk-text-danger uk-link-reset" "uk-link-reset") tagName="tr"}}
+        <td class="uk-text-bold uk-text-default">{{project.name}}</td>
         <td>{{project.customer.name}}</td>
         <td>{{project.billingType.name}}</td>
         <td>{{format-duration-short project.purchasedTime}}</td>
         <td>{{format-duration-short project.spentTime}}</td>
-        <td>{{format-duration-short project.totalTime}}</td>
+        {{#if (gte alertTime project.totalTime._milliseconds)}}
```

Comment:
same as above:
instead of doing this here, please add a computed property on the project model.
this would make the code more readable